src/recipes/8CB1EBAE3B2DA51D/recipe.py

```diff
@@ -0,0 +1,669 @@
+import h5py
+import numpy as np
+
+
+class GeometryExamples:
```

Comment:
The name of this class is not ideal. "Examples" makes you suspect it doesn't parse the file. Why not OFFGeometry or something simple like that?